src/hotspot/share/gc/z/zHeap.inline.hpp

```diff
@@ -43,11 +43,6 @@ inline ReferenceDiscoverer* ZHeap::reference_discoverer() {
   return &_reference_processor;
 }
 
-inline uint32_t ZHeap::hash_oop(uintptr_t addr) const {
-  const uintptr_t offset = ZAddress::offset(addr);
-  return ZHash::address_to_uint32(offset);
```

Comment:
I think removal of the call to `ZHash::address_to_uint32` means this file no longer needs to include zHash.inline.hpp.

Reply:
I think you are right Kim.